docs/adr/adr-010-send-unpublish-emails-for-single-pages.md

```diff
@@ -0,0 +1,74 @@
+
+The email alert API used to send emails when a page in a taxon subscription was unpublished.
+This feature was [removed in Jan 2021][remove-previous-unpublish-emails] for good reasons.
+
+[open-standards-for-government]: https://www.gov.uk/government/publications/open-standards-for-government
+[remove-previous-unpublish-emails]: https://github.com/alphagov/email-alert-api/issues/1572
+
+## Decision
+
+We will reinstate emails when a page is unpublished, but only to users with a single page subscription to that page.
+We will also remove the subscriber list for a single page when it is unpublished to reduce the build up of empty lists in Email Alert API.
+
+### Addressing the concerns from the previous version
+
+The previous version of this feature was removed for several good reasons. We believe readding the unpublish emails only for single page subscriptions addresses these concerns:
+
+#### It only worked for topic taxons
+
+The new unpublished emails will only apply to single page subscriptions. This is a deliberate decision to keep the scope small and allow us to address the other concerns about handling an unpublished page in a taxon.
+
+#### It only worked if the unpublished page was redirected
+
+The new unpublished emails will handle pages that are redirected published in error with or without an alternative URL. This covers the publisher-facing [unpublishing types][unpublishing-types].
+
+#### It wasn't monitored so we didn't know if it was working
+
+Because the new emails will only be for single page subscriptions, it will be easier to test the whole system. We will also use existing methods for templating and sending emails so we can benefit from the monitoring that's already in place.
+
+#### It rendered email using ERB, unlike the rest of the system
+
+The new unpublished emails will use the same method to render emails as the content changes.
+
+#### It removed any subscriber lists that included the page, even those for a different taxon
+
+There's only one single page subscriber list for each page, so it's much easier to know which list to remove when a page is unpublished.
+
+#### There was no evidence it met user needs
+
+Single page notifications are an answer to the user need to stay up to date with specific pieces of government guidance. Especially when the area might be subject to frequent updates or changes.
+
+Our hypothesis is that some users will subscribe to taxons that are broader than their actual area of interest. So they will receive updates about areas they are not interested in, in order to get the few relevant updates to them.
+
+Our hypothesis is that users who subscribe to be notified for major updates, will also want to know if the guidance is unpublished as part of staying up-to-date, and will find it unexpected if guidance they have been monitoring disappears from their account subscriptions without warning.
```

Comment:
We've done user research on this.  Do we have something more concrete than a hypothesis?  eg, can we say "we've seen in user research sessions that being able to subscribe to specific pages of guidance is useful to many users, and meets a different need than subscribing to a group of pages (such as taxon), and that users with these subscriptions expect to be notified if the page is unpublished"?

Reply:
Yeah that last phrase is key, do we actually have UR to that point?
I think it's a fairly solid assumption, but I wasn't sure if it's an expectation we generated or if it came our of UR?

Reply:
I'm sure one of the UCD people would know

Reply:
cool have reworded